### DISCUSSION TITLE
Convert from .avs to .msh, property name between

### DISCUSSION THREAD

**Discussion by cormier7** [nschloe/meshio#1173] [Q&A] [answered]
Hello,

I convert a .avs file to a .msh file ( ascii GMSH 2.2 format)  with the command :

`meshio-convert my_file.avs my_file.msh --output-format gmsh22 --ascii`

Everything work well but the names of the properties in the msh file are between quotation marks:  

$NodeData
1
"property_name"
... 
$EndNodeData

Is there any way to avoid those quotation marks ?

**Comment by gdmcbain** [comment, marked as answer]
Aren't the quotation marks part of MSH 2.2?  In the specification (&sect;[9.3.1 MSH file format version 2 (Legacy)](https://gmsh.info/dev/doc/texinfo/gmsh.html#MSH-file-format-version-2-_0028Legacy_0029)), the example has
```
...
$NodeData
1                      one string tag:
"A scalar view"          the name of the view ("A scalar view")
...
```
